src/anshuKJun20/PalindromeNumber.java

```diff
@@ -0,0 +1,28 @@
+package anshuKJun20;
+
+public class PalindromeNumber {
+	private void findPalindromeNumber(int number) {
+		int number1 = number, add = 0;
+		while (number > 0) {
+			int remainder = number % 10;
+			number = number / 10;
+			add = add * 10 + remainder;
+		}
+		if (add == number1) {
+			System.out.println("Given number " + number1 + " is palindrome.");
+		} else {
+			System.out.println("Given number " + number1 + " is not palindrome.");
+		}
+	}
+
+	public void display() {
```

Comment:
Here no need of extra method.You can directly print details by calling core method.

Reply:
chaanges done.